Makefile

```diff
@@ -37,14 +38,14 @@ KOTS_VERSION = v$(shell echo $(ADMIN_CONSOLE_CHART_VERSION) | sed 's/\([0-9]\+\.
 KOTS_BINARY_URL_OVERRIDE =
 LD_FLAGS = -X github.com/replicatedhq/embedded-cluster/pkg/defaults.K0sVersion=$(K0S_VERSION) \
 	-X github.com/replicatedhq/embedded-cluster/pkg/defaults.Version=$(VERSION) \
-	-X github.com/replicatedhq/embedded-cluster/pkg/defaults.K0sBinaryURL=$(K0S_BINARY_SOURCE_OVERRIDE) \
```

Comment:
this was removed because overridden k0s binaries are still cached in our s3 bucket and we should pull from that in customer installations (as it supports vendor CNAMEs etc)